src/app/components/icons/btc-stamps-icon.tsx

```diff
@@ -0,0 +1,10 @@
+import BitcoinStampImg from '@assets/images/bitcoin-stamp.png';
+import { Box } from '@stacks/ui';
+
+export function BtcStampsIcon() {
+  return (
+    <Box>
+      <img src={BitcoinStampImg} width="36px" />
```

Comment:
You can do `<Box as="img" src={BitcoinStampImg} width="36px" />` here

Reply:
Thanks! I updated it now